sharktuner/README.md

```diff
@@ -65,6 +65,5 @@ pip install --upgrade -r ../requirements-iree-unpinned.txt
 
 ## Examples
 
-Check the `model_tuner` directory for a sample tuner implemented with `libtuner`.
-The [`dispatch example`](model_tuner)
-should be a good starting point for most users.
+For a concrete example, check the `model_tuner` directory for a sample tuner implemented with `libtuner`.
+The [`dispatch example`](model_tuner/README.md) should be a good starting point for most users.
```

Comment:
Could convert `model_tuner` into a link. I also like adding a `/` suffix to directory names in docs to differentiate them from files.
```suggestion
For a concrete example, check the [`model_tuner/` directory](./model_tuner/) for
a sample tuner implemented with `libtuner`. The
[`dispatch example`](model_tuner/README.md) should be a good starting point
for most users.
```